apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -128,6 +153,22 @@ export default class AnimationPickerBody extends React.Component {
   animationCategoriesRendering() {
     const categories = Object.keys(this.props.libraryManifest.categories || []);
     categories.push('all');
+    if (this.props.hideBackgrounds) {
+      return categories
+        .filter(category => category !== 'backgrounds')
```

Comment:
Can we filter categories above, and then the render code would be the same in either case?